res/mock_eth_connector/rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.64.0"
-components = ["rustfmt", "clippy"]
+channel = "nightly-2022-03-15"
```

Comment:
Why so old nightly? Maybe you made a typo in the year? 😄

Reply:
Good catch!